software/firmware/calibrate.py

```diff
@@ -0,0 +1,44 @@
+from machine import Pin, ADC, PWM
+from time import sleep
+
+
+ain = ADC(Pin(26, Pin.IN, Pin.PULL_DOWN))
+cv1 = PWM(Pin(21))
+
+
+def wait_for_voltage(voltage):
+    if voltage != 0:
+        input(f"\nPlease plug {voltage}V into the CV input and then press Enter")
```

Comment:
Yep, I believe that's how it worked with the previous version too.

Reply:
From what I can see the only thing that tied the calibration process to needing a computer/repl connected is the fact that you need to re-write main.py to get into and out of the calibrator. Assuming that we can get out of that need in the future, I think it would be nice if the calibration script could run on its own.

On a related side note I think that the refactor and calibration script should really be two separate PRs.

Reply:
This is also being tracked in #35